docs/UserGuide.md

```diff
@@ -125,6 +128,18 @@ This guide serves to:<br>
 * An example is shown below:
 ![SortExample](images/SortExample.png)
 
+#### Responsive Ui when window resizing
+
+* The Ui will change dynamically when resized.
+
+* When the Ui is resized narrower past a certain point, the Ui will change to a narrow format.
+
+    ![Narrow Ui](images/UiNarrow.png)
+
+* When the Ui is resized wider past a certain point, the Ui will change to a wide format.
+    ![Wide Ui](images/Ui.png)
+
+
```

Comment:
why no gif :(

also, phrasing sounds a bit awkward to me, perhaps consider smth like:
```suggestion

* If the window width is too small, WorkBook will switch to a narrow layout for a better UX.

    ![Narrow Ui](images/UiNarrow.png)

* Otherwise, WorkBook will be in the wide layout which has an extra right panel for displaying tips.
    ![Wide Ui](images/Ui.png)

```
lmk what you think

Reply:
Phrasing changed and the 2 images are now side by side so readers and more easily compare.